px-map-behavior-marker.es6.js

```diff
@@ -345,7 +342,12 @@
       return {
         type: this.type,
         symbol: this.symbolClass,
-        styleScope: this.isShadyScoped() ? this.getShadyScope() : undefined
+        icon: this.icon,
+        styleScope: this.isShadyScoped() ? this.getShadyScope() : undefined,
```

Comment:
We're calling `getComputedStyleValue` three times on three separate style variables. Is that what you're referring to?
I spoke with David about memoizing each of the options (`stroke`, `strokeWidth`, `fill`), where we're calling that method and relying on Polymer's `updateStyles` to clear the cache, but we want to make sure that whenever the style variable changes, the options will be updated accordingly (which I'm not sure we can guarantee if we're relying on `updateStyles`). Happy to hear other ideas.